hermes-schema/src/main/java/pl/allegro/tech/hermes/schema/SchemaCompiler.java

```diff
@@ -1,10 +1,10 @@
 package pl.allegro.tech.hermes.schema;
 
-import pl.allegro.tech.hermes.api.RawSchema;
+import pl.allegro.tech.hermes.api.SchemaMetadata;
 
 @FunctionalInterface
 public interface SchemaCompiler<T> {
 
-    T compile(RawSchema rawSchema);
+    T compile(SchemaMetadata schemaMetadata);
```

Comment:
In my opinion **Raw**Schema fits better for compiler API than Schema**Metadata**. I suggest to restore the previous version of this interface or change `SchemaMetadata` name (maybe `RawSchemaWithMetadata`?). What do you think?

Reply:
Hmm, yes you're right. Compiler should accept RawSchema.